schema/rdl_schema_0.1.json

```
@@ -769,41 +785,52 @@
+          "description": "The media type of this resource, from the open [media_type codelist](https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#media_type). For example a custom binary file has media_type 'application/octet-stream. A geojson file has the media_type 'application/geo+json'.",
           "codelist": "media_type.csv",
           "openCodelist": true,
           "minLength": 1
         },
         "format": {
           "title": "Format",
           "type": "string",
-          "description": "A human-readable descripton of the file format of the resource, taken from the open [data formats codelist]((https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#data_formats).",
+          "description": "A human-readable descripton of the file format of this resource, taken from the open [data formats codelist]((https://rdl-standard.readthedocs.io/en/{{version}}/data_model/codelists/#data_formats).",
           "codelist": "data_formats.csv",
           "openCodelist": true,
           "minLength": 1
         },
         "spatial_resolution": {
           "title": "Spatial resolution",
           "type": "number",
-          "description": "The minimum spatial separation resolvable in the resource, measured in meters.",
+          "description": "The minimum spatial separation resolvable in this resource, measured in meters.",
           "exclusiveMinimum": 0
         },
         "coordinate_system": {
           "title": "Reference coordinate system",
           "type": "string",
-          "description": "The reference co-ordinate system of the dataset version, using the [EPSG codelist](https://epsg.org).",
+          "description": "The reference co-ordinate system of this resource, using the [EPSG codelist](https://epsg.org).",
           "minLength": 1
         },
         "url": {
           "title": "Url",
           "type": "string",
-          "description": "The web address the dataset version can be downloaded from.",
+          "description": "The web address this resource can be downloaded from.",
           "format": "iri"
         },
         "temporal": {
           "title": "Temporal coverage",
           "description": "The period of time covered by this resource.",
           "$ref": "#/$defs/Period"
+        },
+        "temporal_resolution": {
+          "title": "Temporal resolution",
+          "type": "array",
+          "description": "The unit(s) of time covered by this resource, given in ISO 8604 durations.",
+          "items": {
+            "type": "string",
+            "format": "duration"
+          },
+          "minItems": 1,
+          "uniqueItems": true
```

Per https://github.com/GFDRR/rdl-standard/issues/68#issuecomment-1566376926

```suggestion
          "type": "string",
          "description": "The minimum period of time resolvable in this resource, in ISO8601 duration format.",
          "format": "duration"
```